srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -158,12 +159,14 @@ private void parseWeightResults(final Patient patient, final List<String> weight
     {
     	// The last entries are the most recent so we use those.
     	// Get the most recent weight measurement within the already specified range.
-    	final String[] weightLineTokens = weightResults.get(weightResults.size()-2).split("[\\s\\^]+");
+        final List<String> weightLineTokens = Splitter.on(Pattern.compile("[\\s\\^]+"))
```

Comment:
I know you didn't change this, but why are you splitting on this? Is the actual format that you expect from VistA documented anywhere in the code?